tests/storage/merge_test.py

```diff
@@ -236,17 +236,23 @@ def test_block_cow(self, monkeypatch, base, top, expected):
             def fake_measure(image, format=None, output_format=None,
                              backing=True, is_block=False, base=None,
                              unsafe=False):
+                bitmaps = 0
                 # Make sure we are called with the right arguments.
-                assert image == top_vol.getVolumePath()
+                if image == top_vol.getVolumePath():
+                    assert base == base_vol.getVolumePath()
+                    # Top measure returns bitmaps.
+                    bitmaps = 1 * MiB
+                else:
+                    assert image == base_vol.getVolumePath()
+                    assert base is None
```

Comment:
This is very fragile, it will better if the test would set a fake measure function that
returns different value for every volume. One way is to set a function that return values from a mapping from volume id to size of bitmaps.